.github/workflows/build-deploy-pudl.yml

```diff
@@ -141,7 +142,9 @@ jobs:
 
       # Start the VM
       - name: Start the deploy-pudl-vm
-        run: gcloud compute instances start "$GCE_INSTANCE" --zone="$GCE_INSTANCE_ZONE"
+        run: |
+          gcloud compute instances start "$GCE_INSTANCE" --zone="$GCE_INSTANCE_ZONE"
+          gcloud compute ssh "$GCE_INSTANCE" --zone="$GCE_INSTANCE_ZONE" -- "docker image ls catalyst-coop/pudl-etl --digests"
```

Comment:
Added this in an attempt to get the actual hash that's running on the VM.

Reply:
Didn't work due to gcloud authorization issues that I didn't want to work through. Reverted.